localserver.js

```diff
@@ -5,23 +5,29 @@ var fileSystem = require('./server_modules/file_system');
 var args = require('./server_modules/args');
 var views = require('./server_modules/views');
 
-var configs = [require('./server_modules/slave_mode')];
-
-var middlewareLayers = [express.static(fileSystem.resolve('src')),
-                        require('./server_modules/sessions').middleware,
-                        require('./server_modules/auth').basic(args.basicAuthCreds),
-                        require('./server_modules/cors').middleware,
-                        require('./server_modules/cache').middleware,
-                        require('./server_modules/body_builder').middleware,
-                        require('./server_modules/log').middleware];
-
-var routers = [views,
-                fileSystem,
-                require('./server_modules/sessions'),
-                require('./server_modules/teams'),
-                require('./server_modules/scores'),
-                require('./server_modules/challenges')];
-
+var configs = [
+    require('./server_modules/slave_mode'),
+];
+
+var middlewareLayers = [
+    require('morgan')('combined'),
+    express.static(fileSystem.resolve('src')),
+    require('./server_modules/sessions').middleware,
+    require('./server_modules/auth').basic(args.basicAuthCreds),
+    require('./server_modules/cors').middleware,
+    require('./server_modules/cache').middleware,
+    require('./server_modules/body_builder').middleware,
+    require('./server_modules/log').middleware,
```

Comment:
Is there a reason to use both `morgan` and the old `log` module? Shouldn't this be removed and shouldn't the old `server_modules/log.js` file be deleted? What about other server modules that use the `log` module? I think if we move from using our own logging mechanism to using an external one, we should do it in every place the log is used